LEAF_Nexus/templates/view_group.tpl

```diff
@@ -33,9 +35,9 @@
 </div>
 
 <div id="toolbar" class="toolbar_right toolbar noprint">
-    <div id="tools"><h1>Options</h1>
+    <div id="tools"><h1 role="heading" tabindex="0">Options</h1>
```

Comment:
Do labels like this need to be accessible via keyboard?

Reply:
Without the `tabindex`, the header isn't picked up by the screen reader.  The `role="heading"` tells the user that it is a header and not a button to be interacted with.

Reply:
Ok, thanks.